l10n_it_account/__manifest__.py

```diff
@@ -6,8 +6,8 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 {
-    'name': 'Italian Localization - Account',
-    'version': '12.0.1.0.1',
+    'name': 'ITA - Account base',
```

Comment:
Il nome andrebbe in italiano, visto che non è traducibile.
Se vuoi potrebbe eventualmente essere aggiunto un "summary".

Reply:
Grazie, fatto